README.md

```diff
@@ -504,30 +512,60 @@ You can fetch data from Storyblok directly using queries. To add the query:
 3. Create the data retrieval function in the `/apps/.../api/utils` folder using the
    function, type, and hook created by the code gen script.
 
-## GitHub-based blog workflow (Labs blog) 💻
+## GitHub-based blog workflow (Labs and Consulting blogs) 💻
 
-All the **Quansight Labs** blog posts are located inside `apps/labs/post`,
-and therefore, any new posts _must_ be added to this same folder.
+All the blog posts for **both websites** are located inside `./apps/labs/posts`
+and `./apps/consulting/posts`, and therefore, any new posts _must_ be added to
+these same folders.
 
 > **Note**
 > For now, all posts should be
 > contributed using a branch-and-merge strategy within the website repo itself,
 > instead of a fork-and-merge strategy. This may change in the future.
 
-Every post is a `.md` or [`.mdx` file](https://mdxjs.com/docs/using-mdx/). The
-`posts` directory also contains a [`categories.json`
-file](./apps/labs/posts/categories.json) containing the posts categories.
-
-The `categories.json` file is also used for displaying category filters on the
-`/blog` page so after adding a new category, it will also be visible on that
-page.
-
-For more details about `.mdx` please see:
+Every post is a `.md` or [`.mdx` file](https://mdxjs.com/docs/using-mdx/). For
+more details about `.mdx` please see:
 
 - <https://mdxjs.com/>
 - <https://github.com/hashicorp/next-mdx-remote>
 
-### Structure of the blog post
+Each `posts` directory also contains a `categories.json` file
+([Labs file](./apps/labs/posts/categories.json) and
+[Consulting file](./apps/consulting/posts/categories.json)) containing the post
+categories. The formats of these `categories.json` files are currently **not**
+the same:
+
+- For the Labs blog, `categories.json` is an array of case-sensitive strings.
+  These strings are used:
+  - As-is in the rendered category list on the main `/blog` page
+  - In a custom slugified format (spaces replaced with `+`) when filtering
+    `/blog` by category
+    - For example, the category `"Beyond PyData"` is slugified as
+      `"Beyond+PyData`"
+    - This category filter is currently **case-sensitive**—URLs with mismatched
+      case in the `category` query parameter will return **no results**
```

Comment:
```suggestion
  - These strings appear as-is in the list of categories on the main `/blog` page.
  - When filtering `/blog` by category, they appear in a slugified format (spaces replaced with `+`), for example: `/blog?category=Beyond+PyData
  - Since these are case-sensitive, if a blog post includes a category with incorrect case in its metadata, then it will not show up under that category. And in the other direction, if the `category`  query parameter has incorrect case, it will return no results.
```